src/argilla/server/alembic/versions/bda6fe24314e_create_vectors_table.py

```diff
@@ -34,14 +34,17 @@ def upgrade() -> None:
     op.create_table(
         "vectors",
         sa.Column("value", sa.JSON(), nullable=False),
-        sa.Column("vector_settings_id", sa.Uuid(), nullable=False),
+        sa.Column("dataset_id", sa.Uuid(), nullable=False),
```

Comment:
A vector is already associated with a `record_id` so why the need to duplicate that association adding `dataset_id` here?